elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java

```diff
@@ -59,6 +59,8 @@
 class EntityBinding {
 
     private static final List<Method> OBJ_METHODS = Arrays.asList(Object.class.getMethods());
+    private static final List<Class<? extends Annotation>> RELATIONSHIP_TYPES =
```

Comment:
Should we use a set here and just perform a `contains`?

Iterating and comparing a constant list of 4 may be faster than the constant time hashing, though. @wcekan thoughts here?